package.json

```diff
@@ -95,12 +96,16 @@
   "engines": {
     "node": "^18.18 || >=20"
   },
+  "installConfig": {
+    "hoistingLimits": "workspaces"
+  },
   "lavamoat": {
     "allowScripts": {
       "@lavamoat/preinstall-always-fail": false,
       "@keystonehq/bc-ur-registry-eth>hdkey>secp256k1": true,
       "babel-runtime>core-js": false,
-      "simple-git-hooks": false
+      "simple-git-hooks": false,
+      "$root$": false
```

Comment:
Can you explain what this is for?

Reply:
<img width="774" alt="Screenshot 2024-10-22 at 5 53 21 PM" src="https://github.com/user-attachments/assets/e3740468-2743-48ea-9c8d-c982bd95a26b">

Reply:
I don't think we want to set  `"$root$": false` here  https://github.com/MetaMask/metamask-design-system/pull/60/files#r1813382276

Reply:
Updated here https://github.com/MetaMask/metamask-design-system/pull/60/commits/e8d4b6a8de84b19a93639877036aabf2b45dc130